01_fib/brvishnu92/main_test.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"strconv"
+	"testing"
+)
+
+func TestMain(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote("1\n1\n2\n3\n5\n8\n13\n")
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		fmt.Println(actual)
+		t.Errorf("Unexpected output in main()")
+	}
+
+}
+
+func TestMainOutputPositive(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	fib(7)
+
+	expected := strconv.Quote("1\n1\n2\n3\n5\n8\n13\n")
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		fmt.Println(actual)
+		t.Errorf("Unexpected output in main()")
```

Comment:
Fold the last two lines into one with formatted printing:
```go
 t.Errorf("Unexpected output in main(): %v", actual)
```
Error strings should not be capitalised (see [official recommendations](https://github.com/golang/go/wiki/CodeReviewComments#error-strings))